plugins/inputs/gtfs/gfts_test.go

```diff
@@ -0,0 +1,191 @@
+package gtfs_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	gtfsbindings "github.com/MobilityData/gtfs-realtime-bindings/golang/gtfs"
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf/plugins/inputs/gtfs"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func TestGTFS(t *testing.T) {
+	var (
+		testID           = "123"
+		testStopID       = "stop123"
+		testStopSequence = uint32(2)
+		testArrival      = int64(3)
+		testDeparture    = int64(4)
+		testTripID       = "trip123"
+		testVehicleID    = "vehicle123"
+		testRouteID      = "route123"
+		testDirectionID  = uint32(123)
+		testTimestamp    = uint64(100)
+		testLat          = float32(42.42)
+		testLng          = float32(20.20)
+		testBearing      = float32(270)
+		testOdometer     = float64(1000)
+		testSpeed        = float32(60)
+		testVersion      = "v0.0.0"
+		testHeader       = &gtfsbindings.FeedHeader{GtfsRealtimeVersion: &testVersion}
+	)
+
+	for _, test := range []struct {
```

Comment:
Please define the structs and assign to a variable. The iterate over the variable to separate the test-data from the testing logic. This eases reading.